packages/react/src/ActionList/Item.tsx

```diff
@@ -456,7 +428,8 @@ export const Item = React.forwardRef<HTMLLIElement, ActionListItemProps>(
               {slots.blockDescription}
             </Box>
           </ItemWrapper>
-          {!inactive && !menuContext && Boolean(slots.trailingAction) && slots.trailingAction}
+          {!inactive && !loading && !menuContext && Boolean(slots.trailingAction) && slots.trailingAction}
+          {loading === true && <VisuallyHidden>Loading</VisuallyHidden>}
```

Comment:
I think we might need to put this [in the `Box` above](https://github.com/primer/react/blob/main/packages/react/src/ActionList/Item.tsx#L411) so that it is referenced to via `aria-labelledby`, or we could include it in [the `aria-labelledby` list](https://github.com/primer/react/blob/main/packages/react/src/ActionList/Item.tsx#L336) and generate a unique ID for the `VisuallyHidden` component that we could add to the list.

Reply:
Ahh ok I understand.